tests/framework/log/DispatcherTest.php

```diff
@@ -1,10 +1,13 @@
 <?php
 /**
- * @author Carsten Brandt <mail@cebe.cc>
```

Comment:
It looks like some leftovers generated automatically by some IDE. I we want keep file header consistent, this should be removed - `@author` tag can be moved at class-level phpdoc. But I'm not sure if it's worth it - most of test does not have it.